spec/v1.0-draft/statement.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package github.intoto.attestation.spec.v1_0_draft;
+
+import "google/protobuf/struct.proto";
+
+
+option go_package = "github.com/in-toto/attestation/spec/v1.0-draft";
+
+// Proto representation of the in-toto v1.0 Statement.
+message Statement {
+  // Expected to always be "https://in-toto.io/Statement/v1.0"
+  string type = 1 [json_name = "_type"];
+
+  message Subject {
+    string name = 1;
+    map<string, string> digest = 2;
```

Comment:
Added a general comment at the top of each proto that the user is responsible for validating fields.  (Since we don't do validation anywhere else, not just for digest.)